src/distributors/adoptopenjdk-installer.ts

```diff
@@ -84,8 +84,14 @@ export class AdoptOpenJDKDistributor extends JavaBase {
     const jvmImpl = 'hotspot';
     const versionRange = '[1.0,100.0]';
     const encodedVersionRange = encodeURI(versionRange);
+    let release_type = 'ga';
```

Comment:
nitpick: release_type -> releaseType
nitpick2: const releaseType = this.stable ? 'ga' : 'ea'

Reply:
Done. Thank you.